app/components/documents/document-card.hbs

```diff
@@ -56,13 +60,24 @@
                 />
               {{/unless}}
             </Auk::Toolbar::Item>
+            {{#if (and @hasMarkForSignature (not this.loadSignatureRelatedData.isRunning) this.currentSession.isEditor)}}
+              <Auk::Toolbar::Item class="auk-o-flex">
+                <Auk::Button
+                  @skin={{if this.signMarkingActivity  "borderless" "muted"}}
+                  @layout="icon-only"
+                  @icon={{if this.signMarkingActivity  "sign" "no-sign"}}
```

Comment:
De icon is juist, maar de kleur niet.
Hier zou je icon skin "success" aan moeten meegeven, maar in `auk/button.hbs` geven we geen skin mee aan de `Auk::Button::InnerLayout`

Dan zouden we aan de parent button een `@iconSkin` moet meegeven, en dan dan doorgeven aan de innerlayout ` @skin={{@iconSkin}}`, of kan dit anders?
@brenner-company

Reply:
@ValenberghsSven Zoals Johan Ronse dat geïmplementeerd heeft zouden er eigenlijk de skins `muted-borderless` & `success-borderless` moeten gebruikt worden. Nu, die eerste bestaat al, maar die 2de nog niet. Ik maak die laatste nog even aan laat iets weten als die beschikbaar is?

Reply:
`success-borderless` is nu bruikbaar als skin binnen het button component (aanpassingen staan al op de dev-branch).